schema/divisions/defs.yaml

```diff
@@ -89,6 +89,26 @@ description: Common schema definitions for divisions theme
           type: string
           minLength: 1
           pattern: ^(\S.*)?\S$    # Leading and trailing whitespace are not allowed.
+    perspectives:
+      description: Collection of perspectives of single feature.
+      type: object
+      unevaluatedProperties: false
+      required: [holders, includes]
+      properties:
+        holders:
+          description: Array of perspective holders.
+          type: array
+          items: { "$ref": "#/$defs/typeDefinitions/perspective" }
+          minItems: 1
+          uniqueItems: true
+        includes:
```

Comment:
For symmetry, I prefer `effect: "include" | "exclude"` rather than `includes: true | false`.